packages/cli/lib/usageTracking.js

```diff
@@ -1,4 +1,4 @@
-const { trackUsage } = require('@hubspot/cli-lib/api/fileMapper');
+const { trackUsage } = require('@hubspot/local-dev-lib/trackUsage');
```

Comment:
I realized that our [trackUsage in local-dev-lib](https://github.com/HubSpot/hubspot-local-dev-lib/blob/main/lib/trackUsage.ts#L61) never returns anything. There are areas in the CLI that rely on a response being returned (like [here](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli/lib/usageTracking.js#L141)).